tool/tsh/common/tsh.go

```diff
@@ -109,6 +109,8 @@ type CLIConf struct {
 	GopsAddr string
 	// IdentityFile is an argument to -i flag (path to the private key+cert file)
 	IdentityFile string
+	// Compatibility specifies OpenSSH compatibility flags.
```

Comment:
I would mention the flag by name, i.e. `--compat` because without knowing it, another programmer will be trying to figure out what "compatibility" even means.